### PR TITLE
8138588: VerifyMergedCPBytecodes option cleanup needed

### PR DESCRIPTION
**Pull request by coleenp**
This option has been removed in favor of always verifying the bytecodes in debug mode.  Tested with tier1-3.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Testing

|     | Linux x64 | Linux x86 | Windows x64 | macOS x64 |
| --- | ----- | ----- | ----- | ----- |
| Build | ✔️ (5/5 passed) | ✔️ (2/2 passed) | ✔️ (2/2 passed) | ✔️ (2/2 passed) |
| Test (tier1) | ✔️ (9/9 passed) | ✔️ (9/9 passed) | ✔️ (9/9 passed) | ✔️ (9/9 passed) |

### Issue
 * [JDK-8138588](https://bugs.openjdk.java.net/browse/JDK-8138588): VerifyMergedCPBytecodes option cleanup needed

### Reviewers
 * [Harold Seigel](https://openjdk.java.net/census#hseigel) (@hseigel - **Reviewer**)
 * [Daniel D. Daugherty](https://openjdk.java.net/census#dcubed) (@dcubed-ojdk - **Reviewer**)
 * [Serguei Spitsyn](https://openjdk.java.net/census#sspitsyn) (@sspitsyn - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/1137/head:pull/1137`
`$ git checkout pull/1137`

### CODE REVIEW

**Review by hseigel — APPROVED** [review]
Looks good!
Thanks, Harold

File: src/hotspot/share/runtime/globals.hpp [hunk 1]
```diff
@@ -872,10 +872,6 @@ const intx ObjectAlignmentInBytes = 8;
   product(bool, StressLdcRewrite, false,                                    \
           "Force ldc -> ldc_w rewrite during RedefineClasses")              \
                                                                             \
-  /* change to false by default sometime after Mustang */                   \
```
Comment (dcubed-ojdk) [inline src/hotspot/share/runtime/globals.hpp:875]: Well... it's definitely sometime after Mustang/1.6.0... :-)

**Review by dcubed-ojdk — APPROVED** [review]
Thumbs up!
Please make sure this is okay with someone on the Serviceability team.

**Review by sspitsyn — APPROVED** [review]
Hi Coleen,

This looks good to me.
Thank you for taking care about this flag!

Thanks,
Serguei

**Comment by coleenp** [comment]
Thanks Dan, Harold and Serguei.
/integrate

**Pull request closed without merging**